Source/ExcelDnaPack/PackProgram.cs

```diff
@@ -197,7 +197,8 @@ static void Pack(string[] args)
 			string packedName = null;
 			if (integrationPath != null)
 			{
-				packedName = ru.AddAssembly(integrationPath, compress, multithreading);
+				// pdb is not in the nuget package, so chances are we want it if it is present (ExcelDna dev)// pdb is not in the nuget package, so chances are we want it if it is present (ExcelDna dev)	                // pdb is not in the nuget package, so chances are we want it if it is present (ExcelDna dev)// pdb is not in the nuget package, so chances are we want it if it is present (ExcelDna dev)
+				packedName = ru.AddAssembly(integrationPath, compress, multithreading, true);
```

Comment:
I understand this to mean that we would not normally have the `ExcelDna.Integration.pdb` packed, but it a developer want to do so they could set it up so that `ExcelDna.Integration.pdb` is present, and then it would be packed.
I think I agree with this policy, but it should be documented somewhere.

Reply:
Yes, that's exactly the idea : in my case when debugging, I replace the reference to the Nuget package's assembly with a ProjectReference to my local clone of `ExcelDna.Integration.csproj`, in which case `ExcelDna.Integration.pdb` naturally comes along.
Not too sure where this should be documented, though...
And I'll still need to understand how the rebase duplicated the comment without me even noticing it !